packages/react/tsup.config.ts

```diff
@@ -11,5 +13,56 @@ export default defineConfig({
   },
   target: 'esnext',
   sourcemap: true,
-  esbuildPlugins: [styledComponentsPlugin],
+  esbuildPlugins: [
+    styledComponentsPlugin,
+    {
```

Comment:
IMO: Maybe separate to another file or different object, but I failed to find a better location for this.

Reply:
I think it's fine as is